themesource/atlas_core/native/core/helpers/switch.js

```diff
@@ -0,0 +1,29 @@
+import { brand } from "../../variables";
```

Comment:
Why are these changes here?

Reply:
because in the main branch of this repository, there's a missing file _\themesource\atlas_core\web\core\helpers\_slider-color-variant_ so I updated the template with the latest version of Atlas Core (v3.0.8)